doc/optional-components.md

```diff
@@ -114,6 +114,28 @@ spec:
       storage: 500Mi
 ```
 
+## Monitoring with Prometheus
+
+Starting with operator version 1.5.0, you can use [Prometheus](https://prometheus.io/) to monitor Piraeus components.
+The operator will set up monitoring containers along the existing components and make them available as a `Service`.
+
+If you use the [Prometheus Operator](https://prometheus-operator.dev/) it will also set up the `ServiceMonitor`
```

Comment:
```suggestion
If you use the [Prometheus Operator](https://prometheus-operator.dev/), the Piraeus Operator will also set up the `ServiceMonitor`
```

Otherwise "it" could be the Prometheus Operator. Not sure about our capitalization convention for "Piraeus Operator".